services/device2.yaml

```diff
@@ -3,20 +3,3 @@ protocol:
   name: onep
```

Comment:
@sauvainr @colachengexo 
`onep` is removed from device2 since commit: https://github.com/exosite/pegasus_api/commit/e1ead888bde4c4cb300f4d7032b6a401367639fa#diff-7db03cf8489165c7784bc2e151e64085
Should we change it to `onep:v1/http` or `murano:v1/mqtt`?

Reply:
Yes good reminder.

Reply:
You are right! Thanks!